lib/src/request.dart

```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:convert';
+import 'dart:io';
```

Comment:
Sadly, we can't land this. We don't want `dart:io` "things" leaking into the core library. It's meant to work without `dart:io`

Reply:
Thanks for the review. Appreciate it!